packages/eslint-plugin-qwik/package.json

```diff
@@ -32,7 +31,8 @@
   "license": "MIT",
   "main": "dist/index.js",
   "peerDependencies": {
-    "eslint": "^8.57.0"
+    "eslint": "9.15.0",
+    "@types/eslint": "9.6.1"
```

Comment:
This shouldn't be a peer dependency, should it? Is it necessary for the plugin to work?

Reply:
@wmertens 

I guess this should be a peer dependency, because it avoids build errors. 
![image](https://github.com/user-attachments/assets/58b6e620-3c1c-4e65-80dd-ea9e8d09c3eb)

Reply:
Eslint isn’t necessary.
and  `@types/eslint` should be a peer dependency to prevent build error